android/src/com/mapswithme/util/UiUtils.java

```diff
@@ -286,6 +291,15 @@ public static void updateAccentButton(Button button)
                                                                                     : R.attr.accentButtonTextColorDisabled));
   }
 
+  public static Uri getUriToResId(@NonNull Context context, @AnyRes int resId)
+  {
+    final Resources resources = context.getResources();
+    return Uri.parse(ContentResolver.SCHEME_ANDROID_RESOURCE + "://"
```

Comment:
Возможно, сейчас погляжу.

> On Mar 28, 2016, at 3:49 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/util/UiUtils.java <https://github.com/mapsme/omim/pull/2568#discussion_r57568122>:
> 
> > @@ -286,6 +291,15 @@ public static void updateAccentButton(Button button)
> >                                                                                      : R.attr.accentButtonTextColorDisabled));
> >    }
> >  
> > +  public static Uri getUriToResId(@NonNull Context context, @AnyRes int resId)
> > +  {
> > +    final Resources resources = context.getResources();
> > +    return Uri.parse(ContentResolver.SCHEME_ANDROID_RESOURCE + "://"
> Uri.fromParts() не подойдёт?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2568/files/f2dc591f49716076ca1919ceee98db764511cb08#r57568122>